evaluation/edit_api_usecase.md

```diff
@@ -0,0 +1,68 @@
+## Edit An API
+### Primary Actor: Advanced User
+
+### Trigger
+The user wants to change an existing API which is linked to a given function.
+
+### Preconditions
+1. There is at least one (built-in or previously created via the Developer Suite) function available for the user's account.
+2. There is at least one (built-in or previously created via the Developer Suite) API to be editted available for the user's account.
```

Comment:
Again I would say there is one API with the corresponding function.

Reply:
I thought this will be obvious all the time, and often the API which is corresponding got its API first. If an API is needed, WE made sure, so I would not add it as a prerequisite.
We can discuss this if you want in more detail